test/unit/ldp/http/SafeErrorHandler.test.ts

```diff
@@ -0,0 +1,78 @@
+import 'jest-rdf';
+import { DataFactory } from 'n3';
+import type { ErrorHandler } from '../../../../src/ldp/http/ErrorHandler';
+import { SafeErrorHandler } from '../../../../src/ldp/http/SafeErrorHandler';
+import { BasicRepresentation } from '../../../../src/ldp/representation/BasicRepresentation';
+import { NotFoundHttpError } from '../../../../src/util/errors/NotFoundHttpError';
+import { readableToString } from '../../../../src/util/StreamUtil';
+import { HTTP, XSD } from '../../../../src/util/Vocabularies';
+import literal = DataFactory.literal;
+
+describe('A SafeErrorHandler', (): void => {
+  // The error object can get modified by the handler
```

Comment:
That does not seem wise?

Reply:
The comment is wrong, the object gets modified in the tests to test out different scenarios, but not by the handler itself

Reply:
Yeah the `let` makes that clear then; I'd just remove.